tiny_dnn/util/macro.h

```diff
@@ -12,3 +12,11 @@
 #if defined _WIN32 && !defined(__MINGW32__)
 #define CNN_WINDOWS
 #endif
+
+#if defined(_MSC_VER)
+#define CNN_MUSTINLINE __forceinline
+#elif defined(__GNUC__) || defined(__clang__) || defined(__ICC)
+#define CNN_MUST_INLINE __attribute__((always_inline)) inline
+#else
+#define CNN_MUSTINLINE inline
```

Comment:
This one https://github.com/tiny-dnn/tiny-dnn/pull/687#discussion_r121675457
is the cause.